drivers/hmis/app/jobs/hmis/assessment_questions_job.rb

```diff
@@ -79,9 +80,9 @@ def custom_data_element_definitions_by_key
       @custom_data_element_definitions_by_key ||= Hmis::Hud::CustomDataElementDefinition.where(owner_type: 'Hmis::Hud::CustomAssessment').index_by(&:key)
     end
 
-    private def yes_no_nil(bool)
-      return nil if bool.nil?
-
+    private def yes_no(bool)
+      # Nil is intentionally saved as 'No' to match previous behavior.
+      # This will be the case even for questions that were hidden on the page at time of assessment.
       bool ? 'Yes' : 'No'
```

Comment:
It's a little hard to spot in the PR diff, so here's a link to the code showing the behavior on release-115:

https://github.com/greenriver/hmis-warehouse/blob/5cf3a231e0bf944752a1752a620ae7ec8150f1f7/drivers/hmis/app/jobs/hmis/assessment_questions_job.rb#L27-L32